README.md

```diff
@@ -118,6 +119,7 @@ This file contains a number of front-end interview questions that can be used wh
   * If so, what libraries have you used?
 * Explain "hoisting".
 * Describe event bubbling.
+* What is the difference between event bubbling and event capturing.
```

Comment:
We already have a question about event bubbling (which is just above this one). Maybe just focus on event capturing. 

> Describe event capturing.